sql/schema.go

```diff
@@ -105,7 +105,10 @@ func (s *Schema) CheckDBVersion(logger *zap.Logger, db Database) (before, after
 	if err != nil {
 		return 0, 0, err
 	}
-	after = s.Migrations.Version()
+	after = 0
+	if len(s.Migrations) > 0 {
+		after = s.Migrations.Version()
+	}
```

Comment:
`Version` returns 0 if `len(s.Migrations) == 0`

```suggestion
	after = s.Migrations.Version()
```

Reply:
Fixed